### DISCUSSION TITLE
HACS?

### DISCUSSION THREAD

**Discussion by nagyrobi** [travisghansen/hass-pfsense#4] [Ideas]
Do you plan to add this to HACS?

**Comment by travisghansen** [comment]
Yeah, just waiting on this: https://github.com/home-assistant/brands/issues/2903

**Reply by nagyrobi** [reply]
Why don't you just submit a PR with those files?

**Reply by travisghansen** [reply]
I could but didn’t want to go through all the cla headache if one of the devs could copy it quickly. I suppose I’ll need to if they remain unresponsive on the issue.

**Reply by travisghansen** [reply]
Officially included in hacs now FYI.
